src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/legacy/dto/LegacyPeriodLengthTests.kt

```diff
@@ -1,19 +1,33 @@
 package uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.legacy.dto
 
-import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.entity.ChargeEntity
 import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.entity.PeriodLengthEntity
+import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.entity.SentenceEntity
+import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.entity.SentenceTypeEntity
 import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.enum.EntityStatus
 import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.enum.PeriodLengthType
+import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.enum.ReferenceEntityStatus
 import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.jpa.enum.SentenceTypeClassification
 import uk.gov.justice.digital.hmpps.hmppsremandandsentencingapi.legacy.controller.dto.LegacyPeriodLength
-import java.util.UUID
+import java.util.*
 
 class LegacyPeriodLengthTests {
-
   @Test
   fun `return no units when its a life sentence`() {
-    val periodLengthEntity = PeriodLengthEntity(
+    val result = LegacyPeriodLength.from(testPeriodLength, testSentence)
```

Comment:
this test became a little bit more convuluted because I changed the method signature to pass in a SentenceEntity rather than individual attributes from SentenceEntity

Happy to revert if that's the preference